packages/ui/src/components/va-data-table/hooks/useRows.ts

```diff
@@ -18,17 +18,17 @@ export interface TableRow {
   cells: TableCell[];
 }
 
-const buildTableCell = (rowIndex: number, column: TableColumn, source: any): TableCell => ({
-  source,
+const buildTableCell = (rowIndex: number, column: TableColumn, rowData: ITableItem): TableCell => ({
+  rowData,
```

Comment:
```suggestion
  rowData,
  /** @deprecated */
  source: rowData[column.key],
```

This will not force user to refactor existing data-tables.